enterprise/internal/campaigns/syncer.go

```diff
@@ -23,7 +24,10 @@ type ChangesetSyncer struct {
 	// Note that it involves a DB query but no communication with codehosts
 	ComputeScheduleInterval time.Duration
 
-	queue *changesetQueue
+	priorityNotify chan struct{}
+
+	mtx   sync.Mutex
```

Comment:
nit: common name in our codebase
```suggestion
	mu   sync.Mutex
```